worker/uniter/resolver/opfactory.go

```diff
@@ -122,20 +135,20 @@ func (s *resolverOpFactory) wrapHookOp(op operation.Operation, info hook.Info) o
 		op = onPrepareWrapper{op, func() {
 			//on prepare the local status should be made to reflect
 			//that the upgrade process for this united has started.
-			s.LocalState.UpgradeSeriesPrepareStatus = s.RemoteState.UpgradeSeriesPrepareStatus
+			s.LocalState.UpgradeSeriesStatus = s.RemoteState.UpgradeSeriesStatus
```

Comment:
I would declare this anonymous function as a variable and DRY these op declarations up.

Reply:
Given that these just changed, I don't know if it would be worth extracting now. They no longer have text in common.